src/main/java/net/starlark/java/eval/StarlarkInt.java

```diff
@@ -414,6 +439,25 @@ public static StarlarkInt add(StarlarkInt x, StarlarkInt y) {
       return StarlarkInt.of(xl + yl);
     }
 
+    try {
+      long xl = x.toLongFast();
+      long yl = y.toLongFast();
+      long r = xl + yl;
```

Comment:
Please be consistent with names: operands are x and y, result is z.  The suffixes i and l denote int and long.